Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -907,8 +907,11 @@ open class NavigationMapView: UIView {
             let maneuverCoordinate = step.maneuverLocation
             guard step.maneuverType != .arrive else { return }
             
+            let metersPerPointAtLatitude = Projection.metersPerPoint(for: maneuverCoordinate.latitude,
```

Comment:
This variable name no longer reflects the method name; we can just call it `metersPerPoint`.